frontier-chain/runtime/src/lib.rs

```diff
@@ -171,15 +138,15 @@ impl frame_system::Config for Runtime {
 	/// The aggregated dispatch type that is available for extrinsics.
 	type RuntimeCall = RuntimeCall;
 	/// The index type for storing how many extrinsics an account has signed.
-	type Nonce = Nonce;
+	type Nonce = frontier_primitives::Nonce;
 	/// The type for hashing blocks and tries.
-	type Hash = Hash;
+	type Hash = frontier_primitives::Hash;
 	/// The hashing algorithm used.
 	type Hashing = BlakeTwo256;
 	/// The identifier used to distinguish between accounts.
-	type AccountId = AccountId;
+	type AccountId = frontier_primitives::AccountId;
 	/// The lookup mechanism to get account ID from whatever is passed in dispatchers.
-	type Lookup = IdentityLookup<AccountId>;
+	type Lookup = IdentityLookup<frontier_primitives::AccountId>;
```

Comment:
this seems repetitive. it can be used if the import type is used only once, but `AccountId` is everywhere, so can you just import it once in the top of the file?

```rs
use frontier_primitives::*;
```

or 

```rs
use frontier_primitives::{AccountId, Hash, Nonce, ...};
```

Reply:
totally agree. I was following the pattern used here:
* https://github.com/freeverseio/laos/blob/main/ownership-chain/runtime/src/lib.rs
* https://github.com/freeverseio/laos/blob/main/evolution-chain/runtime/src/lib.rs

Reply:
it's not similar to that pattern though. there `primitives::AccountId` is imported only once, it's just type aliased to `AccountId`. here it's imported everywhere its used